src/scripts/components/TimeEntriesList.tsx

```diff
@@ -35,11 +35,23 @@ const getTimeEntryDayGroups = (timeEntries: Array<Array<Toggl.TimeEntry>>): {[da
     }, {})
 };
 
+const TimeEntriesFooter = () => (
+  <Footer>
+    <a target="_blank" href="https://toggl.com/app/timer?utm_source=toggl-button&utm_medium=referral">
+      <Button>
+        See more on <strong>toggl.com</strong>
+      </Button>
+    </a>
+  </Footer>);
+
 interface TimeEntriesListProps {
   timeEntries: Toggl.TimeEntry[][];
   projects: IdMap<Toggl.Project>;
 };
-export default function TimeEntriesList (props: TimeEntriesListProps) {
+export default function TimeEntriesList (props?: TimeEntriesListProps) {
+  if(Object.keys(props).length === 0) {
```

Comment:
TS is throwing an error here

Reply:
PS. I think you don't really need this

Reply:
Line 55 below deals with this, it sets `timeEntries` to an empty array, so you move the if statement after it
and change it something like

```typescript
if (timeEntries.length === 0) {
  return <TimeEntriesFooter />;
}
```

Reply:
Though in this case you might want something like

```typescript
if (timeEntries.length === 0) {
  return (
    <React.Fragment>
      <EmptyState />
      <TimeEntriesFooter />
    </React.Fragment>
  );
}
```

Reply:
updated :raised_hands: